src/components/universal/core/Badge/Badge.recipe.ts

```diff
@@ -0,0 +1,40 @@
+import { defineRecipe } from "@pandacss/dev";
+
+export const badgeRecipe = defineRecipe({
+  className: "badge",
+  description: "The styles for the Badge component",
+  base: {
+    borderRadius: "full",
+    fontWeight: "medium",
+    display: "inline-flex",
+    alignItems: "center",
+    userSelect: "none",
+    whiteSpace: "nowrap",
+    textStyle: "sm",
+    px: 2.5,
+    h: 6,
+    gap: 2,
+    background: "accent.default",
+    color: "accent.fg",
+  },
+  variants: {
+    variant: {
+      subtle: {
+        background: "bg.subtle",
```

Comment:
I would make this `accent.subtle` instead of `bg.subtle`, and remove the border styles for this variant. Just from what I tested manually. Super nit, but those are my thoughts!